app/views/booths/index.html.haml

```diff
@@ -0,0 +1,50 @@
+.container
+  .row
+    .col-md-12.page-header
+      %h1
+        Your booth requests for
+        = @conference.title
+  .row
+    .col-md-12
+      .margin-booth-table
+        %table.table.table-striped.table-hover
+          %thead
+            %th
+              %b ID
```

Comment:
Can you explain your reasoning why you think users should see the ID of their booth request?